### ISSUE TITLE
fuse convolutional and batch_norm weights into one convolutional-layer

### ISSUE THREAD

**Issue by DIAMONDWHILE** [AlexeyAB/yolo2_light#5]
Excuse me
Could you tell me where is the paper on fusion convolution and batch normalization?

**Comment by DIAMONDWHILE** [comment]
Thank you for the explanation.

**Comment by abhigoku10** [comment]
@DIAMONDWHILE @AlexeyAB  can you share the implemented source code of the following idea shared in this link

**Comment by MambaWong** [comment]
> @DIAMONDWHILE @AlexeyAB can you share the implemented source code of the following idea shared in this link

Note: The **convolution layers in YOLO don’t actually use bias**, so b is zero in the above equation. But note that after folding the batch norm parameters, the convolution layers do get a bias term.
https://github.com/AlexeyAB/yolo2_light/blob/ab8be63d1e94cc8980d48050b1a308dd3bada4a7/src/additionally.c#L66-L109
